src/core/map/popover/MapPopover.fixture.tsx

```
@@ -0,0 +1,324 @@
+    if (!this.enabled) return null;
+
+    return (
+      <div style={{ padding: '12px', backgroundColor: 'white', borderRadius: '4px' }}>
+        <h4>Simple Provider</h4>
+        <p>
+          Click position: {mapEvent.point.x}, {mapEvent.point.y}
+        </p>
+        <p>
+          Coordinates: {mapEvent.lngLat.lat.toFixed(4)}, {mapEvent.lngLat.lng.toFixed(4)}
+        </p>
+        <button onClick={onClose}>Close</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)